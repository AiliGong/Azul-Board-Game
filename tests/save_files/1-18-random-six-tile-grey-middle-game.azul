#myformat
3
6
1
a
b
RRBUBLBUYLBROOUROLRLOYBUROBUOBYYRLRRYLYYBUUUYYUYLOBLOYLLRURLYRLUYOUBRBORBBBLLUYOYYLOBUUOLRRYUBLYYOLBOUOBUBOYROOLOBLURRUR
%6
%6
%6
%6
%6
turn 1 B 2
turn 2 L 3
turn 0 B 2
turn 0 R 5
turn 4 O 3
turn 0 U 4
turn 3 Y 1
turn 0 L 3
turn 5 O 3
turn 0 R 5
turn 0 L 4
turn 0 B 1
turn 1 O 1
turn 2 O 1
