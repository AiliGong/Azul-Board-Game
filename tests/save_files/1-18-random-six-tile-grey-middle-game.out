Factories:
0: F Y B U R B U
1:
2:
3: O B Y Y
4: R L R R
5: Y L Y Y

Score for a: 4
Mosaic for a:
1:           O || . . . . . Y 
2:         . . || . . . . . B 
3:       . . . || . . . . . O 
4:     . . L L || . . . . . . 
5:   . . . . . || . . . . . . 
6: . . . . . . || . . . . . . 
broken:

Score for b: 2
Mosaic for b:
1:           O || . . . . . B 
2:         . . || . . . . . . 
3:       . L L || . . . . . . 
4:     . U U U || . . . . . . 
5:   . . . . . || . . . . . R 
6: . . . . . . || . . . . . . 
broken:
